nuxeo-drive-operations/src/main/java/org/nuxeo/drive/operations/NuxeoDriveScrollDescendants.java

```diff
@@ -85,13 +85,13 @@ public Blob run() throws IOException {
     protected Blob writeJSONBlob(ScrollFileSystemItemList scrollFSIList) throws IOException {
         StringWriter writer = new StringWriter();
         JsonFactory factory = new JsonFactory();
-        JsonGenerator jg = factory.createJsonGenerator(writer);
-        jg.setCodec(new ObjectMapper());
-        jg.writeStartObject();
-        jg.writeStringField("scrollId", scrollFSIList.getScrollId());
-        jg.writeObjectField("fileSystemItems", scrollFSIList);
-        jg.writeEndObject();
-        jg.close();
+        try (JsonGenerator jg = factory.createJsonGenerator(writer)) {
```

Comment:
Should that now be `factory.createGenerator` ?

Reply:
Indeed